core/pkg/opaprocessor/processorhandlerutils.go

```diff
@@ -35,7 +35,7 @@ func (opap *OPAProcessor) updateResults(ctx context.Context) {
 
 		// first set exceptions
 		if resource, ok := opap.AllResources[i]; ok {
-			t.SetExceptions(resource, opap.Exceptions, cautils.ClusterName)
+			t.SetExceptions(resource, opap.Exceptions, cautils.ClusterName, opap.AllPolicies.Controls)
```

Comment:
@amirmalka when attempting to achieve something similar (https://github.com/kubescape/kubescape/pull/1080/files#r1099780807) I was in doubt about which is the source of truth for controls here.
You have opap.AllPolicies.Controls (supposedly all configured controls) but also opap.Policies[...].Controls (allegedly all scanned controls). Since the existing unit tests did not hydrate AllPolicies.Controls, I resorted to the latter.

Reply:
@fredbi this is a valid concern.
The `opap.Policies` are the "raw policies", meaning, they are the originally downloaded policies, while the `opap.AllPolicies.Controls` is the [list of controls after removing duplications](https://github.com/kubescape/kubescape/blob/master/core/pkg/opaprocessor/processorhandler.go#L53).

So in this case, we should send the `opap.AllPolicies.Controls` to the function.

Reply:
:ballot_box_with_check: